packages/ui/src/components/Dropdown/CommonItem.tsx

```diff
@@ -5,20 +5,32 @@ import { RefreshIcon } from "../../icons/RefreshIcon.js";
 import { IconProps } from "../../icons/Icon.js";
 
 export type CommonProps = {
-  icon: FC<IconProps>;
+  icon?: FC<IconProps>;
   title: string;
   acting?: boolean;
 };
 
-export const CommonItem: FC<CommonProps> = ({ title, icon, acting }) => {
+const iconDisplay = (icon?: FC<IconProps>, acting?: boolean) => {
+  if (!icon) return null;
   const Icon = acting ? RefreshIcon : icon;
+  return (
+    <Icon
+      size={14}
+      className={clsx(
+        "text-slate-400 group-hover:text-slate-900",
+        acting && "animate-spin"
+      )}
+    />
+  );
+};
+
+export const CommonItem: FC<CommonProps> = ({ title, icon, acting }) => {
   return (
     <div className="rounded px-2 py-1.5 flex items-center gap-2 group hover:bg-slate-100 transition-colors duration-75 cursor-pointer">
-      <Icon
-        size={16}
-        className={clsx("text-slate-900", acting && "animate-spin")}
-      />
-      <div className="text-slate-900 text-sm">{title}</div>
+      {iconDisplay(icon, acting)}
```

Comment:
Misaligned!

I mean, I think there should be padding even if the icon is missing:
<img width="275" alt="Captura de pantalla 2023-06-13 a la(s) 22 19 47" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/2d5f4dc4-d77f-4c1f-8b5c-3fd25d3a3d80">

But also, is this necessary? I think we pass the icon in all cases, currently (I tried to check it with TypeScript). So this might be an artifact from your previous changes.

Reply:
Yea, I think this is an artifact, will review.

About being misaligned, there's a tradeoff there. If it's text only, this is nicer. - That said, we're removing this aspect anyway.